web/src/components/CardForm.js

```diff
@@ -75,12 +78,13 @@ CardForm.propTypes = {
 CardForm.defaultProps = {
   error: false,
   legend: '',
-  onCancel: () => {},
   onSave: false,
   primaryButtonText: ['Save changes', 'Working'],
   sectionName: '',
   success: false,
   working: false
 };
 
-export default CardForm;
+export default withRouter(CardForm);
+
+export { CardForm as plain };
```

Comment:
exporting this as `plain` and then re-importing it as `CardForm` -- what does that do??

Reply:
It lets you test the unwrapped component for better unit isolation.  The default export is wrapped by `withRouter` and I don't want that to interfere with testing just this component.